dist/index.html

```diff
@@ -22,12 +22,12 @@ <h1 class="text-3xl font-bold">Awesome books</h1>
         </nav>
     </header>
     <main class=" flex flex-col -[100%] mx-5 mt-5">
```

Comment:
- [ Required ] Good job on refactoring your Apps code to use `ES-6` syntax, however, it appears that after refactoring your code the App is no longer working. I expected that when I click on the individual links I would be able to see the following.
   
   - [ ] Add book form
   - [ ] Contact details page
   - [ ] A list of books on the homepage
   
-  It's important that while refactoring the code you also make sure that the Application works as expected. Please see the video below and kindly rectify this.

## Video when using the App
https://user-images.githubusercontent.com/16392046/220451172-e2e2be55-7da5-4272-845b-d49bb268e7b9.mp4

Reply:
@Felix45 

Everything works pretty fine at my end and I re-clone the repository and checked
I suggest you kindly checked perhaps you missed something.

Reply:
Hello @Felix45 

Kindly review below link confirm.

Thank you!

https://drive.google.com/file/d/1TSRH-rllmRaTGMvKTrofr61wr4R3sgLJ/view?usp=share_link

Reply:
Hi @shafiuyushawu, 
Are you online, we can set a zoom meeting and have a look at this together. I have checked the project again and it's not working.

Reply:
I set up a call with the student and we found the cause of the bug.